gitops-wg/interested-parties.md

```diff
@@ -107,6 +107,7 @@ Weaveworks | Daniel Lizio-Katzen
 Weaveworks | Brice Fernandes
 Weaveworks | Scott Rigby
 Weaveworks | Lucas Käldström
+Weaveworks | Sebastian Bernheim
```

Comment:
Do you mind adding your GH username? We're asking all new additions. Ideally will also go back and update usernames for everyone already on this list.

```suggestion
Weaveworks | Sebastian Bernheim | sbernheim
```